nixos/modules/services/monitoring/prometheus/exporters.nix

```diff
@@ -224,6 +225,8 @@ in
     services.prometheus.exporters.minio.minioAccessSecret = mkDefault config.services.minio.secretKey;
   })] ++ [(mkIf config.services.rspamd.enable {
     services.prometheus.exporters.rspamd.url = mkDefault "http://localhost:11334/stat";
+  })] ++ [(mkIf config.services.prometheus.exporters.rtl_433.enable {
+    hardware.rtl-sdr.enable = mkDefault true;
```

Comment:
@willibutz Hi, I'm introducing a prom exporter that has a dependency on {a udev ruleset, the `plugdev` group}, which I've abstracted away through this `config.hardware.rtl-sdr` module (available in this PR).

I'm trying to figure out the proper way to enable the `hardware.rtl-sdr` module when the `rtl_433` exporter is enabled too. The surrounding lines are doing the opposite (changing some exporter options when an external module is enabled) so I'm a bit confused what to do – I'm by no mean a NixOS expert. Thanks for your help!

Reply:
There is currently no nice way to do this, I'll need to take another look at moving these defaults in the exporter submodules. So I'd say this is fine, and better than setting this default in the `harware.rtl-sdr` module.

Reply:
ACK, not touching this.